.builds/go-1.10.yml

```diff
@@ -0,0 +1,23 @@
+image: archlinux
+packages:
+  - wget
+sources:
+  - https://github.com/mdlayher/vsock
+environment:
+  GO111MODULE: "on"
+tasks:
+  - install: |
+      wget -q https://dl.google.com/go/go1.10.8.linux-amd64.tar.gz
+      sudo tar -C /usr/local -xzf go1.10.8.linux-amd64.tar.gz
+      sudo ln -s /usr/local/go/bin/go /usr/bin/go
+  - build: |
+      go version
+      cd vsock/
+      go get golang.org/x/lint/golint
+      go get honnef.co/go/tools/cmd/staticcheck
```

Comment:
I'd say it's fine to omit staticcheck on the 1.10.x build due to the use of newer APIs in the tool. We could potentially grab a release build, but I can handle that later.